### PR TITLE
refactor(ES6): Parser.test.js

### PR DESCRIPTION
**Pull request by shubheksha**
**What kind of change does this PR introduce?**
refactor, ES5 => ES6

**Did you add tests for your changes?**
All existing tests pass

**If relevant, link to documentation update:**
N/A

**Summary**
Refactor Parser.test.js to ES6

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: test/Parser.test.js [hunk 1]
```diff
@@ -73,9 +75,9 @@ describe("Parser", function() {
 				})();
 			}, {}
 		],
-		"var definition": [
+		"const definition": [
```
Comment (sokra) [inline test/Parser.test.js]: This should be added as additional test. Tests with `var` should stay too.

File: test/Parser.test.js [hunk 2]
```diff
@@ -85,6 +85,16 @@ describe("Parser", () => {
 				fgh.sub;
 			}, {}
 		],
+		"var definition": [
+			function() {
+				let abc, cde, fgh;
```
Comment (TheLarkInn) [inline test/Parser.test.js]: this is still `let`

File: test/Parser.test.js [hunk 3]
```diff
@@ -126,6 +136,14 @@ describe("Parser", () => {
 				abc: ["test"]
 			}
 		],
+		"renaming with var": [
+			function() {
+				const xyz = abc;
```
Comment (TheLarkInn) [inline test/Parser.test.js]: that is still const, but should be var

**Comment by sokra** [comment]
Thanks

**Pull request merged**